src/components/notifications/SMTP.vue

```diff
@@ -5,6 +5,8 @@
             <input id="hostname" v-model="$parent.notification.smtpHost" type="text" class="form-control" required>
         </div>
 
+        <div class="form-text">{{ $t("HostnameDesc") }}</div>
+
```

Comment:
```suggestion
        <i18n-t tag="div" keypath="Either enter the hostname of the server you want to connect to or localhost if you intend to use a locally configured mail transfer agent" class="form-text">
            <template #localhost>
                <code>localhost</code> 
            </template>
            <template #local_mta>
                <a href="https://wikipedia.org/wiki/Mail_Transfer_Agent" target="_blank">{{ $t("locally configured mail transfer agent") }}</a> 
            </template>
        </i18n-t>
```

Reply:
Looking good to me. I thought I needed to create the translate key. Thanks for the corrections.

Reply:
>  I thought I needed to create the translate key

You did:
In your case `HostnameDesc` was a valid translation key that you created by adding it to `en.json` ^^